### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
https://user-images.githubusercontent.com/24907191/171067848-1330d60c-16bd-480d-9183-05640f454fcc.mov

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,8 +17,40 @@
...
   <header>
+    <h2>Code Journal</h2>
   </header>
   <main>
+    <div class="container">
+      <div data-view="entry-form" class="data-entry-form">
+        <form class="jornal-entry">
+          <h1>New Entry</h1>
+          <div class="row">
+            <div class="column-half column-full">
```
Comment (BrettAlbright) [inline index.html]: these should just be set to column-half, that is why the layout class has that column `100%` width on mobile screens

File: index.html [hunk 2]
```diff
@@ -17,8 +17,40 @@
...
+              <div>
+                <label for="photoUrl">Photo URL
+                  <input type="url" required class="image" id="photoUrl" >
+                </label>
+              </div>
+            </div>
+            <div>
+            </div>
+          </div>
+          <div class="form-inputs">
```
Comment (BrettAlbright) [inline index.html:46]: there should be a separate row and column for the text area.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a few updates for the HTML are needed. 
Also, your data.js file is missing from the pull request
 reach out if you have questions!

**Comment by David-Gonzalez11** [comment]
https://user-images.githubusercontent.com/24907191/171070864-46d0a875-1ba5-49c6-99a5-fcdef3fb033b.mov

Im not sure if i did the dev tools correctly(i didn't see the detached option)  but I tried to hide it as much as i could to still show you the screen.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**